src/cacao.ts

```diff
@@ -39,6 +39,10 @@ export type VerifyOptions = {
    * @param atTime - the point in time the capability is being verified for
    */
   atTime?: Date
+  /**
+   * @param expPhaseOutSecs - Number of seconds that a capability stays valid for after it was expired
+   */
+  expPhaseOutSecs?: number
```

Comment:
```suggestion
  revocationPhaseOutSecs?: number
```
Consistent naming with [js-did/did.ts#L66](https://github.com/ceramicnetwork/js-did/blob/a9763645724409487e2c0d9cf218a06e5ee54ad2/src/did.ts#L66) plz